### PR TITLE
GT-1602 add option to clear parallel language from tool settings

### PR DESCRIPTION
**Pull request by levieggertcru**

### CODE REVIEW

File: godtools/App/Features/LanguagesList/Presentation/LanguagesList/LanguagesListView.swift [hunk 1]
```diff
@@ -54,11 +64,7 @@ struct LanguagesListView_Preview: PreviewProvider {
         
         let selectedLanguageId = "en"
         
-        let viewModel = LanguagesListViewModel(languages: languages, selectedLanguageId: selectedLanguageId, closeTappedClosure: {
-            
-        }, languageTappedClosure: { language in
-            
-        }, deleteTappedClosure: nil)
+        let viewModel = MockLanguagesListViewModel(languages: languages, selectedLanguageId: selectedLanguageId)
```
[the remark this answers is not in the document]
Comment (levieggertcru) [reply godtools/App/Features/LanguagesList/Presentation/LanguagesList/LanguagesListView.swift:67]: Hey @frett yep that's in the preview.  We've been creating base viewModels which hold inputs and outputs an no implementation details that way we can easily mock those for the preview.

[the remark this answers is not in the document]
Comment (frett) [reply godtools/App/Features/LanguagesList/Presentation/LanguagesList/LanguagesListView.swift:67]: We may need to do something similar if we want to make Previews work within Jetpack Compose, I'm just not a huge fan of introducing a bunch of preview/test logic into the main source that gets compiled into release builds. Plus the additional overhead of needing to maintain mock ViewModels in addition to regular ViewModels

[the remark this answers is not in the document]
Comment (frett) [reply godtools/App/Features/LanguagesList/Presentation/LanguagesList/LanguagesListView.swift:67]: any thoughts on the potential of moving previews to a separate source file, e.g. `LanguagesListView+Previews.swift`?

[the remark this answers is not in the document]
Comment (levieggertcru) [reply godtools/App/Features/LanguagesList/Presentation/LanguagesList/LanguagesListView.swift:67]: Hey @frett that makes sense.  There is a lot of overhead in creating those mocks which I'm not fond of either.  We're planning to move mocks into a separate module.  I'm thinking previews can be moved to a separate source file as well.  I will look into that.

**Pull request merged**